### PR TITLE
Feat/fee collector

### PR DESCRIPTION
**Pull request by hochbruj**
https://trello.com/c/aDDiN0F7

### CODE REVIEW

**Comment by xenide** [comment]
I think for the showing balance of the assets, we're supposed to get the balance held by the feeCollector address, instead of the timelock address?

File: components/BalanceTable/index.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const BalanceTable = ({ balances }) => {
                 {formatNumber(balance)}
             </td>
             <td type="num" align="right">
-              {formatDollarAmount(balance * tokens[address].usdPrice)}
+              {tokens[address] && formatDollarAmount(balance * tokens[address].usdPrice)}
```
Comment (KennethAshley) [inline components/BalanceTable/index.js]: sometimes using an `&&` in jsx will produce unlikely results. 

see https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

its suggested to use ternaries rather then &&.

**Review by KennethAshley — CHANGES_REQUESTED** [review]
One response! Nice work though!

File: components/BalanceTable/index.js (hunk 1 above)
Comment (hochbruj) [reply components/BalanceTable/index.js]: Thanks, fixed it

**Comment by hochbruj** [comment]
> I think for the showing balance of the assets, we're supposed to get the balance held by the feeCollector address, instead of the timelock address?

Yes, I replaced it with the fee collector address

File: components/BalanceTable/index.js (hunk 1 above)
Comment (KennethAshley) [reply components/BalanceTable/index.js]: Thanks!

**Review by KennethAshley — APPROVED** [review]

**Pull request merged**